src/genericSchemeRegistry/dxDao.json

```diff
@@ -0,0 +1,135 @@
+{
+  "name": "dxDao",
+  "addresses": {
+    "main": [],
+    "rinkeby": [],
+    "private": []
+  },
+  "actions": [
+    {
+      "id": "updateMasterCopy",
+      "label": "Update MasterCopy",
+      "notes": "https://github.com/gnosis/dx-contracts/blob/master/contracts/base/DxUpgrade.sol#L17",
+      "abi": {
+        "constant": false,
+        "inputs": [
+          {
+            "label": "New Contract Address",
```

Comment:
sorry we cannot do this. The ABi is a strict format, we need to save the input labels apart from that...

Reply:
something like `action.fields = { 
   _masterCopy: { label: "the label" }}
}`